src/config/main.less

```diff
@@ -52,3 +52,7 @@
 @import '../components/NewsCard/newscard.import.less';
 @import '../components/SponsorTag/sponsorTag.import.less';
 @import '../components/Tabs/tabs.import.less';
+
```

Comment:
@OttCS20  This CSS should be wrapped by a parent class to keep it from leaking. Also, you should avoid using `!important` pretty much at all costs, since it escalates the specificity and makes further editing very annoying.